app/controllers/discourse_ai/admin/ai_personas_controller.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module DiscourseAi
+  module Admin
+    class AiPersonasController < ::Admin::AdminController
+      before_action :find_ai_persona, only: %i[show update destroy]
+
+      def index
+        ai_personas =
+          AiPersona.ordered.map { |persona| AiPersonaSerializer.new(persona, root: false) }
+        commands =
+          DiscourseAi::AiBot::Personas::Persona.all_available_commands.map do |command|
+            { id: command.to_s.split("::").last, name: command.name.humanize.titleize }
+          end
+        render json: { ai_personas: ai_personas, meta: { commands: commands } }
+      end
+
+      def show
+        render json: { ai_persona: @ai_persona }
```

Comment:
This should be using `AiPersonaSerializer`, right?

Reply:
actually ... confusingly enough no ... I should probably name stuff "ai_person_class" vs "ai_persona" it makes the code a bit hard to follow ... also thinking of ways of simplifying

Reply:
let me rename the serializer for clarity

Reply:
ok renamed serializer and fixed it